src/macaron/slsa_analyzer/registry.py

```diff
@@ -393,7 +393,12 @@ def scan(self, target: AnalyzeContext, skipped_checks: list[SkippedInfo]) -> dic
                                 current_future.cancel()
                                 self.runner_queue.put(current_runner)
                                 return results
-                        except concurrent.futures.TimeoutError:
+                        except (
+                            concurrent.futures.TimeoutError,
```

Comment:
I'm curious in which scenario did you encounter these newly added exceptions ? or are they suggested from the documentation for the `concurrent.futures` l

Reply:
Some were encountered via testing. But I have added them following the [documentation](https://docs.python.org/fr/3.13/library/concurrent.futures.html#exception-classes).